.github/workflows/ci-pullrequest-main.yml

```
@@ -151,5 +151,11 @@ jobs:
       uses: docker/setup-buildx-action@v2
     
     - name: Docker Build
-      working-directory: src/MockClassifier.Api/
+      working-directory: docker image build src/MockClassifier.Api/ -t mockclassifier
       run: docker image build .
```

This line might to work. Please remove the working-directory part and try this:

```suggestion
      run: docker image build -f src/MockClassifier.Api/Dockerfile ./src -t mockclassifier
```